lib/blemulator/internal/simulation_manager_mixins/client_managing_mixin.dart

```diff
@@ -0,0 +1,9 @@
+part of internal;
+
+mixin ClientManagingMixin on SimulationManagerBase {
+  Future<void> _createClient() async {
+    print("entered create client");
+  }
+
+  Future<void> _destroyClient() async {}
```

Comment:
Don't we want to log that this method was invoked?

Reply:
It's logged from Java.